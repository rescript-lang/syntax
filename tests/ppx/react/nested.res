module Outer = {
  @react.component
  let make = () => {
    module Inner = {
      @react.component
      let make = () => <div />
    }

    <Inner />
  }
}