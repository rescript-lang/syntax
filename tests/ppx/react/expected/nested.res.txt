module Outer = {
  type props = {}
  let make = (_: props) => {
    module Inner = {
      type props = {}

      let make = (_: props) => ReactDOM.jsx("div", {})
      let make = {
        let \"Nested$Outer" = props => make(props)

        \"Nested$Outer"
      }
    }

    React.jsx(Inner.make, {})
  }
  let make = {
    let \"Nested$Outer" = props => make(props)
    \"Nested$Outer"
  }
}
